recipes/scallop/meta.yaml

```diff
@@ -6,11 +6,11 @@ package:
   version: "{{ version }}"
 
 source:
-  url: "https://github.com/Kingsford-Group/scallop/releases/download/v0.10.3/scallop-bioconda-test.tar.gz"
-  sha256: "7148fc961a9e73d9b4a648b26d4c6d013e43d4ab6bf2c075ae09996c01d8cf21"
+  url: "https://github.com/Kingsford-Group/scallop/releases/download/v0.10.4/scallop-0.10.4.tar.gz"
+  sha256: "227d17ba40ee6c4f98967319267c788ced107226106c8f83816307d8af5c3a05"
 
 build:
-  number: 0
+  number: 2
```

Comment:
Can you change this to 1? There's only a build 0. Can you also replace the instances of `0.10.4` with `{{ version }}` so the next upgrade doesn't have the same problem?

Reply:
These changes have been committed. Thanks!